desktop/core/src/desktop/js/reactComponents/PathBrowser/PathBrowser.test.tsx

```diff
@@ -153,8 +156,23 @@ describe('Pathbrowser', () => {
           showIcon
         />
       );
-      const input = screen.queryByDisplayValue('abfs://test/test1');
+      const input = screen.queryByDisplayValue(mockFilePath);
       expect(input).toBeNull();
     });
+
+    it('should show input when edit path button is clicked', async () => {
+      render(
+        <PathBrowser
+          onFilepathChange={onFilepathChangeMock}
+          filePath={mockFilePath}
+          seperator={'/'}
+          showIcon
+        />
+      );
+      const editPathButton = screen.getByTestId('hue-path-browser__edit-path-btn');
```

Comment:
When we write unit tests we should try to verify both the desired start start and the end state. So here I would first check that I cannot see the input field.